rir/src/compiler/opt/scope_resolution.cpp

```diff
@@ -112,8 +112,9 @@ class TheScopeResolution {
             return val;
         };
 
-        auto tryInsertPhis = [&](AbstractPirValue res, BB* bb,
-                                 BB::Instrs::iterator& iter) -> Value* {
+        auto tryInsertPhis = [&](Value* env, AbstractPirValue res, BB* bb,
```

Comment:
It is confusing for me the env as the first parameter. Isn't it there only for unbounds? Maybe only one parameter at the end unboundBB and if it is null we know !allowUnbound?